.env.local

```diff
@@ -91,5 +91,10 @@ WALLET_API_UAT_BASEURL='https://api.uat.platform.pagopa.it'
 SHOW_BARCODE_SCAN_SECTION=NO
 # IT Wallet
 ITW_WALLET_PROVIDER_BASE_URL="https://io-d-itn-wallet-func.azurewebsites.net/"
+ITW_PID_PROVIDER_BASE_URL="https://pre.eid.wallet.ipzs.it"
 ITW_GOOGLE_CLOUD_PROJECT_NUMBER="387904855773"
+ITW_ISSUANCE_REDIRECT_URI="iowallet://cb"
+ITW_ISSUANCE_REDIRECT_URI_CIE="iowalletcie://cb"
```

Comment:
why this are different?

Reply:
The `iowallet` scheme is handled by an [activity](https://github.com/pagopa/io-react-native-login-utils/blob/main/android/src/main/AndroidManifest.xml#L31) created by `io-react-native-login-utils`, that we use for SPID login.
The problem I encountered was that using the same scheme for CieID (which is handled by a browser opened with React Native Linking module) was not redirecting to the app once the login was completed.